langgraph_prototype/tests/integration/test_health_check.py

```
@@ -0,0 +1,248 @@
+"""
+ヘルスチェックとシステム状態確認テスト
+"""
+
+import asyncio
+import sys
+import os
+from unittest.mock import Mock, patch, AsyncMock
```

Ruff (0.12.2)

`unittest.mock.Mock` imported but unused

Remove unused import

(F401)